### PR TITLE
[WIP] enhance: adds csi liveness check for azure csi driver

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

This commit adds csi liveness check to azure csi driver

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/csi-driver/azuredisk-csi-driver/issues/31

**Special notes for your reviewer**:

**Release note**:
```
enhance: adds csi liveness check for azure csi driver
```

### CODE REVIEW

File: pkg/azuredisk/identityserver.go [hunk 1]
```diff
@@ -49,7 +50,7 @@ func (f *Driver) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoReques
 // Currently the spec does not dictate what you should return either.
 // Hence, return an empty response
 func (f *Driver) Probe(ctx context.Context, req *csi.ProbeRequest) (*csi.ProbeResponse, error) {
-	return &csi.ProbeResponse{}, nil
+	return &csi.ProbeResponse{Ready: &wrappers.BoolValue{Value: true}}, nil
```
Comment (ashishranjan738) [inline pkg/azuredisk/identityserver.go]: any suggestion on what are the checks we should perform to conclude the CSI driver to be healthy?

Comment (ashishranjan738) [reply pkg/azuredisk/identityserver.go]: cc @andyzhangx

Comment (andyzhangx) [reply pkg/azuredisk/identityserver.go]: @ashishranjan738 I think we can leave it now as I don't have idea how to implement it now. let's keep this issue open. Thanks.

**Comment by andyzhangx** [comment]
@ashishranjan738 could you also submit a PR for azure file driver? Thanks.
https://github.com/csi-driver/azurefile-csi-driver/issues/28

**Pull request merged**